dependabotAlerts.js

```diff
@@ -39,12 +42,18 @@ function getAlerts(repos) {
       });
       summary["high"] = highAlerts.length;
     }
-    if (mediumAlerts.length > 0 && (summary.critical || summary.high)) {
```

Comment:
I'm not sure why we were excluding medium counts from the summary if we didn't have any high or criticals?